src/AccessibilityTranslator.js

```diff
@@ -58,8 +58,8 @@ export default class AccessibilityTranslator extends React.Component {
    * the accessibility information for all DisplayObjects has been completed(e.g. just after
    * drawing a frame) to make sure that the canvas and accessibility DOM are in sync.
    */
-  update() {
-    this.forceUpdate();
+  update(callback = _.noop) {
+    this.forceUpdate(callback);
   }
```

Comment:
- another attempt at adding a callback, thinking maybe I could use it in a test to capture the React DOM reflow, didn't work

Reply:
- will keep this is in for consumers to use as they see fit